### PR TITLE
Story/configure travis

### PR DESCRIPTION
**Pull request by jicardu**
## Summary

- Add .travis.yml
- Add database.travis.yml
- Add badge of build status

## Trello Card

https://trello.com/c/iEfVjFEg/12-configurar-travisci

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -23,7 +23,9 @@ default: &default
 
 development:
   <<: *default
-  database: rs-rails_development
+  database: wbooks_api
```
Comment (marcegl91) [inline config/database.yml]: Please follow Rails convention over how the database should be named.

File: .gitignore [hunk 2]
```diff
@@ -19,3 +19,4 @@
 /yarn-error.log
 
 .byebug_history
+.env
```
Comment (sebasdeldi) [inline .gitignore]: EOF

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  - bundle exec rspec spec -fd
+  - bundle exec rubocop app spec --format simple
+
+#notifications:
+#  email:
+#    recipients:
+#      - rails-bootstrap-dev@wolox.com.ar
+
+cache:
+  bundler: true
```
Comment (sebasdeldi) [inline .travis.yml]: EOF

File: config/database.travis.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+test:
+  adapter: postgresql
+  encoding: unicode
+  database: travis_wbooks_api_test
+  username: travis_wbooks_api
+  password: travis_wbooks_api
```
Comment (sebasdeldi) [inline config/database.travis.yml]: EOF

**Pull request merged**